img2dataset/downloader.py

```diff
@@ -348,26 +458,63 @@ def download_one_file(url_list):
             timeout=timeout,
         )
 
+        # Start a W&B run
```

Comment:
I notice a new run will be created for each file
Probably we the run should be started in the main function to have only one run for all the files no ?

Reply:
It actually doesn't start a new run if it's already existing.
I wanted to create it as late as possible in case there are other errors before (input file, format, etc).

Reply:
Ok
Related to that, it's not introduced by your change but it might be a good time to change it : the counters are reset for each new file, as well as the status dict.
If there are many small input files, this could be confusing
Do you think it could make sense to put these counters outside of the "one file" function ?
Just need to be careful ram doesn't explode with the status dict, but from your tests it seems to be ok